pkg/investigations/clustermonitoringerrorbudgetburn/clustermonitoringerrorbudgetburn.go

```diff
@@ -0,0 +1,87 @@
+// Package clstermonitoringerrorbudgetburn contains remediation for https://issues.redhat.com/browse/OCPBUGS-33863
+package clustermonitoringerrorbudgetburn
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	configv1 "github.com/openshift/api/config/v1"
+	investigation "github.com/openshift/configuration-anomaly-detection/pkg/investigations"
+	k8sclient "github.com/openshift/configuration-anomaly-detection/pkg/k8s"
+	"github.com/openshift/configuration-anomaly-detection/pkg/ocm"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	uwmMisconfiguredSL = ocm.ServiceLog{
+		Severity:     "Major",
+		Summary:      "Action required: review user-workload-monitoring configuration",
+		ServiceName:  "SREManualAction",
+		Description:  "Your cluster's user workload monitoring is misconfigured: the user-workload-monitoring-config ConfigMap in the openshift-user-workload-monitoring namespace is misconfigured. Please review your configuration alongside the product documentation: https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/monitoring/configuring-the-monitoring-stack#.",
+		InternalOnly: false,
+	}
+)
+
+func Investigate(r *investigation.Resources) error {
+	ctx := context.Background()
+	scheme := runtime.NewScheme()
```

Comment:
Minor: I'd find the `investigate` function much easier to parse if we moved the whole schema registration part into `k8sclient.New()`.

Reply:
The k8sclient needs different schemas based on what we want to do with it, I'm not sure how much of an overhead it is to just add all schema initializations for all functions in that one place. Do you know what the standard practice is?

Reply:
yeah, i was wondering about that as well. Most of the times that i've seen this schema initializtion was during initialization of an operator (e.g. [aao](https://github.com/openshift/aws-account-operator/blob/master/main.go#L65)), where this is only called once. 
I wouldn't think that adding unneccessary schemes has a noticable performance impact, especially for this use case, but I don't have any proof to back that up :)

Reply:
I'm happy with moving it, it definitely makes things cleaner. We can always move it back out later. 

Thanks for the feedback!